lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Screens/BaseClasses/PostDetail/LMFeedBasePostDetailViewModel.swift

```diff
@@ -26,7 +26,7 @@ public class LMFeedBasePostDetailViewModel {
         self.postID = postID
         self.commentList = []
         self.currentPage = 1
-        self.pageSize = 10
+        self.pageSize = 20
```

Comment:
Was it intentional or was for testing purpose ?

Reply:
It was intentional